godtools/ViewControllers/MenuViewController.swift

```diff
@@ -210,41 +216,43 @@ extension MenuViewController {
     
     fileprivate func openHelp() {
         let url = URL(string: "http://godtoolsapp.com")
-        self.delegate?.openWebView(url: url!, title: "help".localized)
+        self.delegate?.openWebView(url: url!, title: "help".localized, analyticsTitle: "Help")
     }
     
     fileprivate func contactUs() {
         if MFMailComposeViewController.canSendMail() {
             sendEmail(recipient: "support@godtoolsapp.com", subject: "Email to GodTools support")
         } else {
             let url = URL(string: "http://www.godtoolsapp.com/contact.html")
-            self.delegate?.openWebView(url: url!, title: "contact_us".localized)
+            self.delegate?.openWebView(url: url!, title: "contact_us".localized, analyticsTitle: "Contact Us")
         }
     }
     
     fileprivate func shareGodToolsApp() {
         let textToShare = [ "share_god_tools_share_sheet_text".localized ]
         let activityViewController = UIActivityViewController(activityItems: textToShare, applicationActivities: nil)
         self.present(activityViewController, animated: true, completion: nil)
+        sendScreenViewNotification(screenName: "Share App")
```

Comment:
I would rather track this as an event later since it's not truly a screen within the app.